mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -45,6 +45,8 @@ def _mailgun_event_type_mapping(self):
             'bounced': 'hard_bounce',
             'dropped': 'reject',
```

Comment:
I see in https://documentation.mailgun.com/en/latest/quickstart-events.html#events there's no `dropped` nor `bounced` statuses, but yes there's a `stored` state that we don't have here. Should we fix that now BTW?

Reply:
They come from the hooks: https://documentation.mailgun.com/en/latest/api-webhooks-deprecated.html?highlight=bounced#bounces-parameters

Reply:
Ah ok. And about `stored`?

Reply:
> Mailgun has stored an incoming message

I'd say it doesn't apply here